test-cases/longevity/longevity-encryption-at-rest-200GB-6h.yaml

```diff
@@ -29,3 +29,19 @@ authorizer: 'CassandraAuthorizer'
 
 pre_create_schema: True
 scylla_encryption_options: "{ 'cipher_algorithm' : 'AES/ECB/PKCS5Padding', 'secret_key_strength' : 128, 'key_provider': 'LocalFileSystemKeyProviderFactory', 'secret_key_file': '/tmp/secret_key'}"
+
+# enable system_info_encryption, config kmip_hosts
+append_conf: |
+  system_key_directory: /etc/encrypt_conf/
```

Comment:
this is too hacky and opens a "window" for injecting erroneous configuration into Scylla YAML. 
Currently, no need to change this, but will need to refactor in future

Reply:
Any graceful suggestion / idea to edit multiple-lines in scylla.yaml from SCT? and it can be configurable in submitting jobs from Jenkins?